templates/toil-instance.yaml

```diff
@@ -72,6 +72,29 @@ Parameters:
   PublicRsaKey:
     Description: Public RSA key for instance
     Type: String
+  ToilAppliance:
+    Description: Docker image to use for the toil instances.
+    Type: String
+    Default: 'quay.io/tessthyer/amp-rnaseq_reprocess:toil-3.20.0'
+  ClusterName:
+    Description: Name of the cluster, will be applied to all instances associated with it.
+    Type: String
+  ClusterLeaderNodeType:
+    Description: EC2 instance type for the leader node in the toil cluster.
+    Type: String
+    Default: 'm5.large'
+  ClusterLeaderStorage:
+    Description: Amount of disk in Gigabytes to reserve for the cluster leader.
```

Comment:
would it make sense to specify MIN and MAX possible value for this?

Reply:
I don't think a max, at least, not at this time. The two ways the toil cluster can work are to either gather outputs back to the leader, in which case we don't want to be too restrictive on the value, or to write to a bucket. I don't think it's necessarily advisable to force people to always write to a bucket. I'd prefer to address these questions in a later refactor, and after we've had more time to run big workflows, and after I talk to the workflow users more.

As for min, leaderstorage for toil defaults to 50. I'll just set it to that.